src/node/cli.ts

```diff
@@ -751,25 +743,9 @@ export const shouldOpenInExistingInstance = async (args: UserProvidedArgs): Prom
  * Convert our arguments to VS Code server arguments.
  */
 export const toVsCodeArgs = async (args: DefaultedArgs): Promise<CodeServerLib.ServerParsedArgs> => {
```

Comment:
I think we will need to remove `workspace` and `folder` from the VS Code side because right now they are required arguments.

Reply:
This is the only blocker btw, the rest of my comments are suggestions/optional.

Reply:
Whoops! That's why `yarn lint` is unhappy with me. Good call!